scripts/leapfrog/generate_resources.py

```diff
@@ -0,0 +1,322 @@
+#! /usr/bin/env python
+from argparse import ArgumentParser
+from ipaddress import IPv4Network
+from itertools import chain, izip, repeat, islice
+from json import dumps
+import os
+from time import sleep
+
+from keystoneauth1.identity import v3
+from keystoneauth1 import session as session_
+from cinderclient import client as cinder_client
+from glanceclient import client as glance_client
+from neutronclient.v2_0 import client as neutron_client
+from novaclient import client as nova_client
+
+CLIENT_VERSION = "2"
+
+
+def get_conf():
+    parser = ArgumentParser()
+    parser.add_argument("--servers", type=int, default=1)
+    parser.add_argument("--volumes", type=int, default=0)
+    parser.add_argument("--networks", type=int, default=1)
+    parser.add_argument("--infinite-quotas", action="store_true")
+    args = parser.parse_args()
+    if args.servers > args.networks:
```

Comment:
The request was to keep it simple, there are obviously multiple ways these things can be combined however it only needs to support one at this time, the pattern I chose was to create separate networks for each server. Is this going to cause an issue with the testing?